website/docs/reference/warehouse-profiles/hive-profile.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Apache Hive Profile"
+id: "hive-profile"
+---
+
+## Overview of dbt-hive
+
+**Maintained by:** Cloudera    
+**Author:** Cloudera    
+**Source:** [Github](https://github.com/cloudera/dbt-hive)    
+**dbt Cloud:** Currently un-supported    
+**dbt Slack channel** [Link to channel](https://getdbt.slack.com/)     
+
+
+![dbt-hive stars](https://img.shields.io/github/stars/cloudera/dbt-hive?style=for-the-badge)
+
+## Connection Methods
+
+dbt-hive can connect to Apache Hive and Cloudera Data Platform clusters.
+
+The [Impyla](https://github.com/cloudera/impyla/) library is used to establish connections to Hive.
+
+Two transport mechanisms are supported:
+- binary
+- HTTP(S)
+
+The default mechanism is `binary`. To use HTTP transport, use the boolean option `use_http_transport: [true / false]`.
+
+## Authentication Methods
+
+dbt-hive supports three authentication mechanisms:
```

Comment:
We have onmy mentioned 2 authentication methods,

Reply:
Yes, we haven't implemented kerberos yet